### ISSUE TITLE
Failed to connect or change directory  jenkins.plugins.publish_over.BapPublisherException: Failed to change to remote directory [/]

### ISSUE THREAD

**Issue by dmytroKarataiev** [rcgroot/jenkins-publish-over-dropbox-plugin#5]
I added an API token and when I tried to add a Dropbox location I received the following error:

Failed to connect or change directory
jenkins.plugins.publish_over.BapPublisherException: Failed to change to remote directory [/]

When I put the path field blank - I receive a success message, but another error:
Remote directory must be an absolute path. (Starting with a /)

**Comment by vekexasia** [comment]
@dmytroKarataiev did you manage to fix the issue?

**Comment by dmytroKarataiev** [comment]
@vekexasia yeah, I did. You need to create a remote directory first. It doesn't accept / for some reason (it's in the Settings for a Jenkins server, not for a project).
In the end I decided not to use this plugin at all, as I needed to publish 30+ flavors and Dropbox after about 100 uploads per day stops uploading them.

**Comment by vekexasia** [comment]
@dmytroKarataiev what do you use instead? If I may ask?

**Comment by dmytroKarataiev** [comment]
@vekexasia publish all of them on Fabric and Google Play as Alpha, don't keep apk files other than on the Jenkins server itself.
